NewForce-Cohort-6/tabloidfullstack-stuffing
# SB/32-Unsubscribe

# Description

Functionality added for a user to unsubscribe from an author.

# Testing Instructions

`git checkout SB/32-Unsubscribe`

1. Login with any user
2. Navigate to Posts via navbar
3. Navigate to the post details page by clicking the tile of any post
4. Use the "Subscribe to Author" link to subscribe if you haven't yet
5. Navigate to Home via the navbar
6. Ensure that the author's posts that you subscribed to are listed
7. Click on a post title to view the post details page and click unsubscribe 
8. Verify that you've been unsubscribed from that author by either querying the database or ensuring that you cannot see the authors posts in the Home post lists.

# Checklist:
